examples/dcll_hcpb_examples/dcll_radial_build_example/dcll_radial_build.py

```diff
@@ -0,0 +1,123 @@
+import openmc
+from radial_build_tools import ToroidalModel, RadialBuildPlot
+from dcll_json_to_openmc import makematerial
+import json
+
+def build_dcll_radial_model():
+    major_radius = 800
+    minor_radius_z = 114
+    minor_radius_xy = 114
+    scoring_layer_thickness = 0.5
+
+    path_to_cross_sections='/filespace/l/lygre/cnergresearch/data/endfb-viii.0-hdf5/cross_sections.xml'
+    materials=openmc.Materials.from_xml("mixedMaterialsDCLL_libv1.xml")
+    materials.cross_sections=path_to_cross_sections
```

Comment:
Is there a reason you aren't using the `OPENMC_CROSS_SECTIONS` environment variable? If you truly want to set the path using this method to allow for more control for future users, would an `xs_path` command-line argument work as opposed to hard-coding the path?